code/modules/surgery/plastic_surgery.dm

```diff
@@ -4,10 +4,37 @@
 	steps = list(
 		/datum/surgery_step/incise,
 		/datum/surgery_step/retract_skin,
+		/datum/surgery_step/insert_plastic,
 		/datum/surgery_step/reshape_face,
 		/datum/surgery_step/close,
 	)
 
+//insert plastic
+/datum/surgery_step/insert_plastic
+	name = "insert plastic (plastic)"
+	implements = list(
+		/obj/item/stack/sheet/plastic = 100,
+		/obj/item/food/meat/slab = 100,
+		/obj/item/stack/sheet/meat = 100)
+	time = 32
+	preop_sound = 'sound/effects/blobattack.ogg'
+	success_sound = 'sound/effects/attackblob.ogg'
+	failure_sound = 'sound/effects/blobattack.ogg'
+
+/datum/surgery_step/insert_plastic/preop(mob/user, mob/living/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	display_results(
+		user,
+		target,
+		span_notice("You begin to insert [tool] into the incision in [target]'s [parse_zone(target_zone)]..."),
+		span_notice("[user] begins to insert [tool] into the incision in [target]'s [parse_zone(target_zone)]."),
+		span_notice("[user] begins to insert [tool] into the incision in [target]'s [parse_zone(target_zone)]."),
+	)
+	display_pain(target, "You feel something inserting just below the skin in your [parse_zone(target_zone)].")
+
+/datum/surgery_step/insert_plastic/success(mob/user, mob/living/target, target_zone, obj/item/tool, datum/surgery/surgery, default_display_results)
+	. = ..()
+	tool.Destroy()
```

Comment:
never call destroy directly, use qdel - or in t his case, probably `stack.use(num_here)`.